scripts/build-android.js

```diff
@@ -34,26 +36,28 @@ const copyOutputPath = path.resolve(process.cwd(), "react-native", "android", "s
 const buildTypes = ["Debug", "Release", "RelWithDebInfo", "MinSizeRel"];
 let architectures = ["x86", "armeabi-v7a", "arm64-v8a", "x86_64"];
 const optionDefinitions = [
-    { name: 'arch', type: validateArchitectures, multiple: false, description: "Build only for a single architecture" },
-    { name: 'changes', type: Boolean, defaultValue: false, multiple: false, description: "Build changes only" },
-    { name: 'buildType', type: validateBuildType, defaultValue: "Release", multiple: false, description: "CMAKE_BUILD_TYPE: Debug, Release, RelWithDebInfo, MinSizeRel" },
+    { name: "arch", type: validateArchitectures, multiple: false, description: "Build only for a single architecture" },
+    { name: "changes", type: Boolean, defaultValue: false, multiple: false, description: "Build changes only" },
+    { name: "build-type", type: validateBuildType, defaultValue: "Release", multiple: false, description: "CMAKE_BUILD_TYPE: Debug, Release, RelWithDebInfo, MinSizeRel" },
```

Comment:
I need to make this clear to my teammates. Will probably have to change with https://github.com/realm/realm-js/issues/3708 too.